src/notebooks/student_submissions.ipynb

```diff
@@ -2,136 +2,306 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 105,
    "metadata": {},
    "outputs": [],
    "source": [
     "import pandas as pd\n",
+    "import lms_utilities as lms\n",
+    "import os\n",
     "import matplotlib\n",
     "\n",
-    "%matplotlib inline\n",
-    "matplotlib.style.use('ggplot')"
+    "# removes decimal places from float datatype\n",
+    "pd.options.display.float_format = '{:,.0f}'.format"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 106,
    "metadata": {},
    "outputs": [],
    "source": [
-    "df_submission = pd.read_csv(\"..\\studentsubmissions.csv\")\n",
-    "df_student = pd.read_csv(\"..\\students.csv\")\n",
-    "df_merged = df_student.merge(df_submission, how='inner', on=('userId', 'courseId'))"
+    "# set the directory path for the output files from google classroom extractor\n",
+    "csv_path = os.path.join(\"..\", \"google-classroom-extractor\", \"data\", \"ed-fi-udm-lms\")"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 107,
    "metadata": {},
    "outputs": [],
    "source": [
-    "df_out = df_merged[[\"profile.name.fullName\",\"profile.emailAddress\"]]\n",
-    "df_out = df_out.drop_duplicates()\n",
-    "df_out = df_out.set_index('profile.emailAddress')"
+    "# pull user data into a dataframe\n",
+    "users_df = lms.get_all_users(csv_path)\n",
+    "\n",
+    "# pull section data into a dataframe\n",
+    "sections_df = lms.get_all_sections(csv_path)\n",
+    "\n",
+    "# pull assignment data into a dataframe\n",
+    "assignments_df = lms.get_all_assignments(csv_path, sections_df)\n",
+    "\n",
+    "# pull submission data into a dataframe\n",
+    "submissions_df = lms.get_all_submissions(csv_path, assignments_df)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 108,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# merge user and submission data\n",
+    "user_submissions_df = pd.DataFrame()\n",
+    "user_submissions_df = pd.merge(users_df, submissions_df, how='inner', left_on=('SourceSystemIdentifier','SourceSystem'), right_on=('LMSUserIdentifier','SourceSystem'))"
```

Comment:
`LMSUserIdentifier` should only be in the database, not in the CSV file. So this should be 
```suggestion
    "user_submissions_df = pd.merge(users_df, submissions_df, how='inner', left_on=('SourceSystemIdentifier','SourceSystem'), right_on=('LMSSourceSystemUserIdentifier','SourceSystem'))"
```